lib/taskserver/configure_taskserver.dart

```diff
@@ -254,15 +324,36 @@ class TaskrcWidget extends StatefulWidget {
 }
 
 class _TaskrcWidgetState extends State<TaskrcWidget> {
+  final TextEditingController _taskrcContentController =
+      TextEditingController();
   bool hideKey = true;
 
+  Storage get storage => widget.storage;
+
   @override
   Widget build(BuildContext context) {
     var color =
         AppSettings.isDarkMode ? Colors.white : Palette.kToDark.shade200;
 
     Server? server;
     Credentials? credentials;
+
+    void setContent() async {
+      final clipboardData = await Clipboard.getData(Clipboard.kTextPlain);
+      _taskrcContentController.text = clipboardData?.text ?? '';
+      if (_taskrcContentController.text.isNotEmpty) {
+        storage.taskrc.addTaskrc(_taskrcContentController.text);
```

Comment:
off issue but maybe add some checks to adding a task? Right now it just writes to a file if I'm not wrong

Reply:
Hey, sorry for the late response.
I have added the commit for the above case.

1. It fetches the clipboard data and assigns it to the `_taskrcContentController` text field.
2. If the clipboard data is not empty, then it adds the data to the taskrc file.
3. It reads the contents of the taskrc file and creates a `Taskrc` object by parsing the contents.
4. Then It checks if the `Taskrc` object contains both the server and credentials.
5. If both are present,then the `server` and `credentials` variables are both updated using `setState(){}`.
6. In cases where the server or credentials are missing,a snackbar error message is displayed .
7. Similarly, if there is an error reading the taskrc file, an appropriate error message is shown.

Also, I have improved the _showStatistics() widget.
Now it will show a progress dialog while the stats are being fetched and parsed.